site/schemas/create-consequence-stops.schema.ts

```diff
@@ -0,0 +1,130 @@
+        invalid_type_error: "Id must be a number",
+    }),
+    atcoCode: z.string({
+        required_error: "Atco code is required",
+        invalid_type_error: "Atco code must be a string",
+    }),
+    naptanCode: z
+        .string({
+            required_error: "Naptan code is required",
+            invalid_type_error: "Naptan code must be a string",
+        })
+        .optional(),
+    commonName: z.string({
+        required_error: "Common name is required",
+        invalid_type_error: "Common name must be a string",
+    }),
+    street: z
+        .string({
+            required_error: "Street is required",
+            invalid_type_error: "Street must be a string",
+        })
+        .optional(),
+    indicator: z.string().optional(),
+    bearing: z
+        .string({
+            required_error: "Bearing is required",
+            invalid_type_error: "Bearing must be a string",
+        })
+        .optional(),
+    nptgLocalityCode: z
+        .string({
+            required_error: "Nptg locality code is required",
+            invalid_type_error: "Nptg locality code must be a string",
+        })
+        .optional(),
+    localityName: z
+        .string({
+            required_error: "Locality name is required",
+            invalid_type_error: "Locality name must be a string",
+        })
+        .optional(),
+    parentLocalityName: z
+        .string({
+            required_error: "Parent locality name is required",
+            invalid_type_error: "Parent locality name must be a string",
+        })
+        .optional(),
+    longitude: z.string({
+        required_error: "Longitude is required",
+        invalid_type_error: "Longitude must be a string",
+    }),
+    latitude: z.string({
+        required_error: "Latitude is required",
+        invalid_type_error: "Latitude must be a string",
+    }),
+    stopType: z
+        .string({
+            required_error: "Stop type is required",
+            invalid_type_error: "Stop type must be a string",
+        })
+        .optional(),
+    busStopType: z
+        .string({
+            required_error: "Bus stop type is required",
+            invalid_type_error: "Bus stop type must be a string",
+        })
+        .optional(),
+    timingStatus: z
+        .string({
+            required_error: "Timing status is required",
+            invalid_type_error: "Timing status must be a string",
+        })
+        .optional(),
+    administrativeAreaCode: z
+        .string({
+            required_error: "Administrative area code is required",
+            invalid_type_error: "Administrative area code must be a string",
+        })
+        .optional(),
+    status: z
+        .string({
+            required_error: "Status is required",
+            invalid_type_error: "Status must be a string",
+        })
+        .optional(),
+});
+
+export type Stop = z.infer<typeof stopsImpactedSchema>;
+
+export const createConsequenceStopsSchema = z.object({
```

Comment:
In my last ticket I moved the consequence schemas into a shared consequence.schema.ts so that we can create a discriminated union of all the different consequence schemas, the contents of this file can be moved into there and we can make use of the baseConsequenceSchema rather than redefine the shared properties

Reply:
done